src/visualization/types/Gauge/view.tsx

```diff
@@ -29,32 +29,52 @@ const GaugeChart: FC<Props> = ({result, properties}) => {
     decimalPlaces,
   } = properties
 
+  // delete after testing code
+  const {theme} = useContext(AppSettingContext)
   const currentTheme = theme === 'light' ? GAUGE_THEME_LIGHT : GAUGE_THEME_DARK
 
-  return (
-    <LatestValueTransform table={result.table} allowString={false}>
-      {latestValue => (
-        <AutoSizer>
-          {({width, height}) => (
-            <div className="gauge">
-              <Gauge
-                width={width}
-                height={height}
-                colors={colors}
-                prefix={prefix}
-                tickPrefix={tickPrefix}
-                suffix={suffix}
-                tickSuffix={tickSuffix}
-                gaugePosition={latestValue}
-                decimalPlaces={decimalPlaces}
-                theme={currentTheme}
-              />
-            </div>
-          )}
-        </AutoSizer>
-      )}
-    </LatestValueTransform>
-  )
+  if (isFlagEnabled('useGiraffeGraphs')) {
+    const config: Config = {
+      table: result.table,
+      layers: [
+        {
+          type: 'gauge',
+          prefix: prefix,
+          suffix: suffix,
+          tickPrefix: tickPrefix,
+          tickSuffix: tickSuffix,
+          decimalPlaces: decimalPlaces,
+          gaugeColors: colors,
+        },
+      ],
+    }
+    return <Plot config={config} />
+  } else {
```

Comment:
since this returns, you don't even need the `else` here

Reply:
i agree with this. imo, dropping the else and putting the return outside looks better.